cart/urls.py

```diff
@@ -0,0 +1,23 @@
+"""wePractice URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+from cart.views import OrderCreateView, OrderListView
```

Comment:
```suggestion
from cart.views import (
    OrderCreateView,
    OrderListView
)
```
서로 다른 클래스는 행으로 구분하여 import 해주세요

Reply:
수정 완료했습니다!